src/schema.mjs

```diff
@@ -125,6 +125,47 @@ export const jsonrpc = {
   ],
 };
 
+export const ipfs = {
+  type: "object",
+  properties: {
+    type: {
+      type: "string",
+      enum: ["ipfs"],
+    },
+    commissioner: {
+      type: "string",
+    },
+    version: {
+      type: "string",
+    },
+    options: {
+      type: "object",
+      properties: {
+        timeout: {
+          $comment: "temporal unit is milliseconds",
+          type: "integer",
+        },
+        url: { type: "string" },
+        gateway: {
+          type: "string",
+          pattern: "^.+ipfs/$",
```

Comment:
I think gateways always start with `https://`

Reply:
We can add this. Mainly I added the regular expression to make sure that the extraction-worker doesn't break.

The extraction-worker concatenates gateway with CID.

Reply:
I added this